pkg/sql/split.go

```diff
@@ -30,12 +30,11 @@ import (
 type splitNode struct {
 	optColumnsSlot
 
-	tableDesc                catalog.TableDescriptor
-	index                    catalog.Index
-	rows                     planNode
-	run                      splitRun
-	expirationTime           hlc.Timestamp
-	testAllowSplitAndScatter bool
```

Comment:
`testAllowSplitAndScatter` does not need to exist because `knobs.AllowSplitAndScatter` can be checked later on.